src/app/shared/object-list/my-dspace-result-list-element/claimed-search-result/claimed-search-result-list-element.component.html

```diff
@@ -4,7 +4,14 @@
                                [showSubmitter]="showSubmitter"
                                [badgeContext]="badgeContext"
                                [workflowItem]="workflowitem$.value"></ds-themed-item-list-preview>
-
+  <!-- Display duplicate alert, if feature enabled and duplicates detected -->
```

Comment:
<img width="798" alt="Screenshot 2024-02-01 at 14 01 48" src="https://github.com/DSpace/dspace-angular/assets/8408850/e81805fa-55f8-45dc-b41b-e8008a545f2b">

Duplicate alert needs to be aligned with other lines.

At his stage, the item already is claimed, so the alert should not tell users to "Claim and edit this item to see details." but just to "Edit this item to see details.".

Reply:
Fixed in both claimed and pooled task template